README.md

```diff
@@ -64,24 +64,49 @@ chmod +x checks.sh
 
 This will run static analysis and return any errors found. It will also reformat the code in-place to meet our style guidelines, which are based on the LLVM guidelines. If you would like to check if the code meet style guideline _without_ it being updated in place, then the flag `--formatting` can be passed to the script.
 
-# Social Media
-<a href="https://www.instagram.com/daftengineers/"><img src="https://user-images.githubusercontent.com/10051310/219481632-10430e66-73dc-400e-a046-dc9d7dd9f3a9.svg" height=80px></a>
-<a href="https://twitter.com/DaftEngineers"><img src="https://user-images.githubusercontent.com/10051310/219482103-8422c45f-ca60-4918-b5b8-6de79d0add22.png" height=80px></a>
-<a href="https://www.youtube.com/@daftengineers"><img src="https://user-images.githubusercontent.com/10051310/219484585-2eb87c95-5951-428b-b3a1-7ac846e40f65.png" height=80px></a>
-<a href="https://www.tiktok.com/@daftengineers"><img src="https://user-images.githubusercontent.com/10051310/220791551-141ee0cc-34ef-47b8-bbcb-701461f88851.png" height=80px></a>
-
 # Documentation
 
-Documentation for main can be [found here](https://daft-engineers.github.io/vocoder/).
+Documentation for main can be [found here](https://daft-engineers.github.io/vocoder/annotated.html).
+
+If you want to generate the documentation for yourself, the `doxygen` and `graphviz` packages are required. These can be installed with the following command on ubuntu:
 
-If you want to generate the documentation for yourself, from the root of the project, the command 
+```$
+sudo apt install doxygen graphviz
```

Comment:
Can you not just run "doxygen" from the project root?

Reply:
if you've got it installed, yeah

Reply:
He means you don't need to do `doxygen Doxyfile`, but it doesn't matter imo. It works as is